lib/timetable-form/Form.tsx

```diff
@@ -84,6 +87,16 @@ export const Form = (props: React.HTMLAttributes<HTMLDivElement>) => {
     reader.readAsText(file);
   }, []);
 
+  useEffect(() => {
+    fetchLPAs()
+      .then((lpas) => {
+        setLPAs(lpas.entities.map((lpa) => lpa.name));
+      })
+      .catch((error: Error) => {
+        console.error(error.message);
+      });
+  }, []);
```

Comment:
We can also rewrite this without using `then` and `catch` (doing it this way also doesn't seem to bring in the correct types):

```suggestion
  useEffect(() => {
    (async () => {
      try {
        const lpas = await fetchLPAs();
        setLPAs(lpas.entities.map((lpa) => lpa.name));
      } catch (error) {
        console.error(error instanceof Error ? error.message : error);
      }
    })();
  }, []);
```

The function could also be named if you didn't want to keep it anonymous:

```suggestion
  useEffect(() => {
    const loadLpas = async () => {
      try {
        const lpas = await fetchLPAs();
        setLPAs(lpas.entities.map((lpa) => lpa.name));
      } catch (error) {
        console.error(error instanceof Error ? error.message : error);
      }
    };
    loadLpas();
  }, []);
```